04_numeronym/luliangLi/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestTransfer(t *testing.T) {
+
+	testCases := map[string]struct {
+		input string
+		want  string
+	}{
+		"normal": {input: "luliangLi", want: "l7i"},
```

Comment:
More comprehensive test cases covering edges cases should be included. E.g. "Web Development" as per wiki page. Special characters? (should that be included as part of the string?)